demo-java/app/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,6 @@
   <uses-permission android:name="android.permission.INTERNET" />
   <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
   <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
-  <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Comment:
Removed since this is already added by the Places SDK's manifest file and so the demo app has this permission implicitly.

Reply:
We should also remove L22 and L23 as well since the SDK adds those permissions too, right?